src/gc.c

```diff
@@ -628,9 +630,8 @@ gc_mark_children(mrb_state *mrb, mrb_gc *gc, struct RBasic *obj)
     {
       struct REnv *e = (struct REnv*)obj;
 
-      if (MRB_ENV_ONSTACK_P(e) && e->cxt && e->cxt->fib) {
-        mrb_gc_mark(mrb, (struct RBasic*)e->cxt->fib);
-      }
```

Comment:
I was checking again and found a period when this section did not exist.

```console
% git log -L627,640:src/gc.c 22518b5b789293e7ddad7a94def44039937d8e8c
```

There is no mention of a commit message, but it corresponds to a revert of commit b6598e052fda9a27692156ab8b973ce6e3374ee3.